### ISSUE TITLE
Add. 14,631, foll. 17–44 | https://bl.vuexistapps.us/manuscript/380 (Part2)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#995]
### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- The note that appears at the end of the Content section ("They are numbered with red letters on the margin")  also seems related to Phys. Des. 

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

**Comment by mikey-tang16** [comment]
Question
Regarding the note under content, I think it makes more sense to have it stay there, as it is more closely connected with the "content" about the anecdotes?
@wlpotter

**Comment by wlpotter** [comment]
Agreed that it should stay with contents. But let's also add a decoDesc (after the handDesc) with a decoNote, with a type of "rubrication" saying that the anecdotes are numbered with red letters on the margins. See https://bl.syriac.uk/documentation/guidelines#decodesc for more info and an example, but we can also walk through this one if needed

**Comment by mikey-tang16** [comment]
I added decoNote,  but let's double check. Also, the decoNote now sort of repeats the note under content. Should we delete the first sentence in Content note and just keep the second?

**Comment by mikey-tang16** [comment]
Leave the repeat as is
